typescript/examples/model-context-protocol-cdp-server/claude_desktop_config-local.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
nit: can the filename just be `claude_desktop_config.json`? looking to avoid the different casing in the filename – otherwise we could go with `claude_desktop_config_local.json`